src/main/java/com/iridium/iridiumskyblock/upgrades/SizeUpgrade.java

```diff
@@ -19,6 +19,6 @@ public SizeUpgrade(int money, int crystals, int size) {
     @JsonIgnore
     @Override
     public List<Placeholder> getPlaceholders() {
-        return Collections.singletonList(new Placeholder("size", String.valueOf(size)));
+        return Arrays.asList(new Placeholder("dimensions", String.valueOf(size)), new Placeholder("size", String.valueOf(size)));
```

Comment:
Whats the point of this?

Reply:
may old users still remember it as "dimension" instead "size"
So i've just added an alternative

Reply:
I mean I dont see this being a massive issue since its default as size, but like :shrug: